.github/workflows/build_and_test/action.yml

```diff
@@ -46,3 +46,13 @@ runs:
         name: golden-test-failures
         path: test/golden/failures
         if-no-files-found: ignore
+
+    - name: 💾 Cache Gradle
+      uses: actions/cache@v4
+      with:
+        key: gradle
+        path: /home/runner/.gradle
+
+    - name: 🔨 Build Apk
+      shell: bash
+      run: flutter build apk --debug
```

Comment:
Should it build profile instead?

Reply:
Though, I suppose it would require a release signature, which is currently WIP in https://github.com/nt4f04uNd/sweyer/pull/118/files

Reply:
I think we could leave it debug for now, and create an issue to make it profile

Reply:
I chose to use a debug build to avoid requiring any signing keys, and to avoid spending time on minification.

Since the point of this is mainly to proof that the apk compiles, I'm not sure what benefit switching to profile would bring. Could you please elaborate on why it would be better?

Reply:
> Since the point of this is mainly to proof that the apk compiles, I'm not sure what benefit switching to profile would bring. Could you please elaborate on why it would be better?

It's useful to manually test the app from the PR. It gives you the closes build to production, while still leaving some of the debug tooling

Reply:
Ah, I didn't consider that we would want to use it for manual testing, I just wanted a way to verify that the apk can be build. This makes sense, I changed it to profile.